src/easymotion/easymotion.ts

```diff
@@ -131,33 +131,35 @@ export class EasyMotion {
   private static getSvgDataUri(code: string, backgroundColor: string, fontFamily: string, fontColor: string,
     fontSize: string, fontWeight: string): vscode.Uri {
 
-      // Clear cache if the backgroundColor has changed
-      if (this.setBackgroundColor !== backgroundColor) {
-        this.svgCache = {};
-        this.setBackgroundColor = backgroundColor;
-      }
+    // Clear cache if the backgroundColor has changed
+    if (this.setBackgroundColor !== backgroundColor) {
```

Comment:
Not a huge deal but the cached resources will be out of date whenever you change a setting BESIDES the background color, so tuning the colors is a bit complicated. A clean restart of vscode handles this. It might be a good idea to check some other things here and clear the cache in the case of any main changes now that they are all configurable at runtime